common/services/prismic/articles.js

```diff
@@ -200,7 +281,7 @@ export async function getArticles(req: ?Request, {
     Prismic.Predicates.any('document.type', ['articles', 'webcomics'])
   ].concat(predicates), {
     orderings,
-    graphQuery,
+    graphQuery: graphQuery,
```

Comment:
What no shortcut? Just curious.

Reply:
It was just as I was trying to replace some of the whitespace characters to give us more room in the URL (Prismic sends the Graph query on the URL for caching reasons).

It was moved to here:
https://github.com/wellcometrust/wellcomecollection.org/pull/3566/files#diff-dccecb3db8753d09dbf462271a9a8ba5R169